### DISCUSSION TITLE
How can I use it offline?

### DISCUSSION THREAD

**Discussion by okuc** [goniszewski/grimoire#90] [Q&A]
Thanks for developing such a great app, I really like it. But I'm having a problem now, I deployed it to a server on my intranet, but when I'm not working on my intranet, I can't access my bookmarks that I've bookmarked.
I would like to ask if there is a corresponding solution in this case, e.g.: provide local cache, provide offline client?

**Comment by tamaskan** [comment]
i think the easiest solution would be to use a vpn to connect to your intranet

**Comment by goniszewski** [comment]
Hello @okuc! Grimoire needs constant internet connection at this time. The support for offline access may be added later with PWA support, but I don't know when it will happen.

I apologize for the delayed response. Your message went under my radar somehow 😞
